docs/detections/prebuilt-rules/downloadable-packages/0-13-3/prebuilt-rule-0-13-3-web-application-suspicious-activity-unauthorized-method.asciidoc

```diff
@@ -0,0 +1,48 @@
+[[prebuilt-rule-0-13-3-web-application-suspicious-activity-unauthorized-method]]
+=== Web Application Suspicious Activity: Unauthorized Method
+
+A request to web application returned a 405 response which indicates the web application declined to process the request because the HTTP method is not allowed for the resource
```

Comment:
```suggestion
A request to web application returned a 405 response which indicates the web application declined to process the request because the HTTP method is not allowed for the resource.
```